src/material-examples/table-pagination/table-pagination-example.ts

```diff
@@ -0,0 +1,103 @@
+import {Component, ViewChild} from '@angular/core';
+import {DataSource, MdPaginator} from '@angular/material';
+import {BehaviorSubject} from 'rxjs/BehaviorSubject';
+import {Observable} from 'rxjs/Observable';
+import 'rxjs/add/operator/startWith';
+import 'rxjs/add/observable/merge';
+import 'rxjs/add/operator/map';
+
+@Component({
+  selector: 'table-pagination-example',
+  styleUrls: ['table-pagination-example.css'],
+  templateUrl: 'table-pagination-example.html',
+})
+export class TablePaginationExample {
+  propertiesToDisplay = ['userId', 'userName', 'progress', 'color'];
```

Comment:
Since selection will be used in a later example, it might be confusing if theyre related. How about displayedColumns?